multiLevelCoSurrogates/CandidateArchive.py

```diff
@@ -38,7 +37,7 @@ def __init__(self, ndim: int, fidelities=None):
 
 
     @classmethod
-    def from_multi_fidelity_function(cls, multi_fid_func, ndim: int=None):
+    def from_multi_fidelity_function(cls, multi_fid_func):
```

Comment:
For a [`classmethod`](https://docs.python.org/3/library/functions.html#classmethod), this is a common way to name them, since they are typically called explicitly from the class itself. So in this case: `CandidateArchive.from_multi_fidelity_function()`. 

See also [dict.fromkeys()](https://docs.python.org/3/library/stdtypes.html#dict.fromkeys) and [itertools.chain.from_iterable()](https://docs.python.org/3/library/itertools.html#itertools.chain.from_iterable) 🙂 .

Reply:
ahh, didn't know about this. Learned something new today. Thank you!